service/collector/chart_resource.go

```diff
@@ -10,23 +13,25 @@ import (
 	"github.com/giantswarm/chart-operator/service/controller/chartconfig/v7/key"
 )
 
-const (
-	chartNameLabel     = "chart_name"
-	channelNameLabel   = "channel_name"
-	releaseNameLabel   = "release_name"
-	releaseStatusLabel = "release_status"
-)
-
 var (
 	chartConfigDesc *prometheus.Desc = prometheus.NewDesc(
 		prometheus.BuildFQName(Namespace, "", "chartconfig_status"),
 		"Managed charts status.",
 		[]string{
-			chartNameLabel,
-			channelNameLabel,
-			releaseNameLabel,
-			releaseStatusLabel,
-			namespaceLabel,
+			labelChart,
+			labelChannel,
+			labelRelease,
+			labelReleaseStatus,
+			labelNamespace,
+		},
+		nil,
+	)
+
+	cordonExpireTimeDesc = prometheus.NewDesc(
+		prometheus.BuildFQName(Namespace, "", "cordon_expire_time"),
```

Comment:
I think it would be good to add `_seconds` here https://prometheus.io/docs/practices/naming/